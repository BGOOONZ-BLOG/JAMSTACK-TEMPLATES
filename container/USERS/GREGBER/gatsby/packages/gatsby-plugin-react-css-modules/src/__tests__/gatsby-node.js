const { onCreateBabelConfig } = require(`../gatsby-node`)

describe(`gatsby-plugin-react-css-modules`, () => {
  let actions
  beforeEach(() => {
    actions = {
      setBabelPlugin: jest.fn(),
    }
    onCreateBabelConfig({ actions }, { plugins: jest.fn() })
  })

  it(`should add babel-plugin-react-css-modules plugin to babel config`, () => {
    expect(actions.setBabelPlugin.mock.calls[0]).toMatchSnapshot()
  })

  it(`should not pass plugins to babel plugin config`, () => {
    expect(actions.setBabelPlugin.mock.calls[0].plugins).not.toBeDefined()
  })
})
