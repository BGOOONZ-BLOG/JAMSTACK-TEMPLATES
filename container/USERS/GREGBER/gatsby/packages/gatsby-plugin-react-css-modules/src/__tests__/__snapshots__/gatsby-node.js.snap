// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gatsby-plugin-react-css-modules should add babel-plugin-react-css-modules plugin to babel config 1`] = `
Array [
  Object {
    "name": "babel-plugin-react-css-modules",
    "options": Object {
      "generateScopedName": "[name]--[local]--[hash:base64:5]",
      "webpackHotModuleReloading": true,
    },
  },
]
`;
